civitai-downloader/src/cli.py

```
@@ -0,0 +1,168 @@
+    if base_model:
+        click.echo(f"Base models: {', '.join(base_model)}")
+
+
+@cli.command()
+@click.argument('model_id', type=int)
+@click.option('--version', '-v', help='Specific version to show')
+@click.option('--images', '-i', is_flag=True, help='Show preview images')
+@click.pass_context
+def show(ctx, model_id: int, version: Optional[str], images: bool):
+    """Show detailed information about a model."""
+    config = ctx.obj['config']
```

Ruff (0.12.2)

Local variable `config` is assigned to but never used

Remove assignment to unused variable `config`

(F841)